packages/block-editor/src/components/font-family/index.js

```diff
@@ -25,7 +20,7 @@ export default function FontFamilyControl( {
 		fontFamilies = blockLevelFontFamilies;
 	}
 
-	if ( isEmpty( fontFamilies ) ) {
+	if ( ! fontFamilies || fontFamilies.length === 0 ) {
```

Comment:
`fontFamilies` is being used as an array a few lines below already, so making sure it's not nullish and then checking for the array length is a safe bet.